src/main/java/com/exadel/discountwebapp/exception/WebExceptionHandler.java

```diff
@@ -25,6 +25,12 @@ public String entityNotFoundException(EntityNotFoundException ex) {
         return ex.getMessage();
     }
 
+    @ExceptionHandler(EntityAlreadyExistsException.class)
+    @ResponseStatus(value = HttpStatus.NOT_FOUND)
```

Comment:
Semantically it is more preferable to use HttpStatus.CONFLICT status here or something similar, rather than NOT_FOUND

Reply:
status changed